src/main/java/com/onehouse/config/models/configv1/MetadataExtractorConfig.java

```diff
@@ -47,4 +48,9 @@ public enum JobRunMode {
     CONTINUOUS,
     ONCE
   }
+
+  public enum UploadStrategy {
+    BLOCKING,
```

Comment:
nit: maybe we can rename this to BLOCK_ON_INCOMPLETE_COMMIT or something similar to be more clear about what we are blocking on?